bin/compile

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env bash
 # bin/compile <build-dir>
+BUILD_DIR=$1
 GHOSTSCRIPT_VERSION="$(cat $BUILD_DIR/.ghostscript_version)"
 
 echo "-----> Installing $GHOSTSCRIPT_VERSION using build directory $1"
```

Comment:
Let’s make it $BUILD_DIR instead of $1 everywhere

Reply:
done @RomanMukhin